### PR TITLE
Simplify the XML-JSON conversion BBE

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/3118

### CODE REVIEW

File: examples/xml-to-json-conversion/xml_to_json_conversion.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# XML JSON conversion
+
+The `xmldata` library provides API to perform conversion from XML to JSO.
```
Comment (daneshk) [inline examples/xml-to-json-conversion/xml_to_json_conversion.md:3]:
```suggestion
The `xmldata` library provides API to perform conversion from XML to JSON.
```

File: examples/xml-from-json-conversion/xml_from_json_conversion.bal [hunk 2]
```diff
@@ -16,11 +16,4 @@ public function main() returns error? {
     // and the default `arrayEntryTag` (i.e., `root`).
     xml? xmlData = check xmldata:fromJson(jsonObject);
     io:println(xmlData);
```
Comment (daneshk) [inline examples/xml-from-json-conversion/xml_from_json_conversion.bal:18]: Shall we rename `jsonObject` to `jsonValue`

File: examples/xml-to-json-conversion/xml_to_json_conversion.bal [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import ballerina/io;
+import ballerina/xmldata;
+
+public function main() returns error? {
+    xml xmlData = xml `<Store id="AST">
```
Comment (daneshk) [inline examples/xml-to-json-conversion/xml_to_json_conversion.bal:5]:
```suggestion
    // Creates a XML value.
    xml xmlValue = xml `<Store id="AST">
```

**Review by daneshk — APPROVED** [review]

File: examples/xml-from-json-conversion/xml_from_json_conversion.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# XML JSON conversion
```
Comment (daneshk) [inline examples/xml-from-json-conversion/xml_from_json_conversion.md:1]: is the title correct? this name should be consistent with XML-Record BBEs

File: examples/index.json [hunk 5]
```diff
@@ -2866,8 +2866,15 @@
...
-                "name": "XML JSON conversion",
-                "url": "xml-json-conversion",
+                "name": "XML to JSON conversion",
+                "url": "xml-to-json-conversion",
+                "verifyBuild": true,
+                "verifyOutput": true,
+                "isLearnByExample": false
+            },
+            {
+                "name": "XML from JSON conversion",
```
Comment (daneshk) [inline examples/index.json:2876]: Need to change here

File: examples/xml-from-json-conversion/xml_from_json_conversion.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# JSON to XML conversion
+
+The `xmldata` library provides API to perform conversion from JSON to XML.
```
Comment (praneesha) [inline examples/xml-from-json-conversion/xml_from_json_conversion.md:3]:
```suggestion
The `xmldata` library provides an API to perform conversions from JSON to XML.
```

File: examples/xml-from-json-conversion/xml_from_json_conversion.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+For more information on the underlying module, see the [xmldata module](https://lib.ballerina.io/ballerina/xmldata/latest/).
```
Comment (praneesha) [inline examples/xml-from-json-conversion/xml_from_json_conversion.md:5]:
```suggestion
For more information on the underlying module, see the `[xmldata` module](https://lib.ballerina.io/ballerina/xmldata/latest/).
```

File: examples/xml-from-json-conversion/xml_from_json_conversion.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+::: code xml_from_json_conversion.bal :::
+
+To run this sample use the `bal run` command.
```
Comment (praneesha) [inline examples/xml-from-json-conversion/xml_from_json_conversion.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```

File: examples/xml-to-json-conversion/xml_to_json_conversion.bal [hunk 9]
```diff
@@ -0,0 +1,19 @@
+import ballerina/io;
+import ballerina/xmldata;
+
+public function main() returns error? {
+    // Creates a XML value.
```
Comment (praneesha) [inline examples/xml-to-json-conversion/xml_to_json_conversion.bal:5]:
```suggestion
    // Creates an XML value.
```

File: examples/xml-to-json-conversion/xml_to_json_conversion.description [hunk 10]
```diff
@@ -0,0 +1,3 @@
+// The `xmldata:fromJson()` and `xmldata:toJson()` functions are used to do the conversions between JSON and XML.
+// For more information on the underlying module,
+// see the [xmldata module](https://lib.ballerina.io/ballerina/xmldata/latest/).
```
Comment (praneesha) [inline examples/xml-to-json-conversion/xml_to_json_conversion.description:3]:
```suggestion
// see the [`xmldata` module](https://lib.ballerina.io/ballerina/xmldata/latest/).
```

File: examples/xml-to-json-conversion/xml_to_json_conversion.md [hunk 11]
```diff
@@ -0,0 +1,11 @@
+# XML to JSON conversion
+
+The `xmldata` library provides API to perform conversion from XML to JSON.
```
Comment (praneesha) [inline examples/xml-to-json-conversion/xml_to_json_conversion.md:3]:
```suggestion
The `xmldata` library provides an API to perform conversions from XML to JSON.
```

File: examples/xml-to-json-conversion/xml_to_json_conversion.md [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+
+For more information on the underlying module, see the [xmldata module](https://lib.ballerina.io/ballerina/xmldata/latest/).
```
Comment (praneesha) [inline examples/xml-to-json-conversion/xml_to_json_conversion.md:5]:
```suggestion
For more information on the underlying module, see the [`xmldata` module](https://lib.ballerina.io/ballerina/xmldata/latest/).
```

File: examples/xml-to-json-conversion/xml_to_json_conversion.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+
+::: code xml_to_json_conversion.bal :::
+
+To run this sample use the `bal run` command.
```
Comment (praneesha) [inline examples/xml-to-json-conversion/xml_to_json_conversion.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```

File: examples/xml-to-json-conversion/xml_to_json_conversion.out [hunk 14]
```diff
@@ -0,0 +1,2 @@
+bal run xml_json_conversion.bal
```
Comment (praneesha) [inline examples/xml-to-json-conversion/xml_to_json_conversion.out:1]:
```suggestion
$ bal run xml_json_conversion.bal
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**